### ISSUE TITLE
[NEW]: Swyftx

### ISSUE THREAD

**Issue by Dennisblofenin** [molly/ftx-contagion#15]
### Details

Swyftx (cryptocurrency exchange)

Exposure: Claim of no exposure.

What actions have they taken: 
TL;DR: Investors took $48 million out of the business, fired 90 staff (after firing 70 earlier in the year) and are now going back to the market to raise more money "to sure up its balance sheet".

1. "Australian cryptocurrency exchange Swyftx has announced it will lay off 40 per cent of its workforce, around 90 people, due to a worsening downturn in digital asset markets following the collapse of overseas exchange FTX." 

2. "Swyftx had paid out $48.5 million in dividends earlier this year before it tapped the market for funding."

3. "Swyftx has run to market for a short-term capital injection, seeking to shore up its balance sheet and ability to fund longer-term growth plans."

### Links

1. https://www.smh.com.au/business/companies/worst-case-scenario-local-crypto-exchange-swyftx-lays-off-40-percent-of-staff-20221205-p5c3mu.html
2. https://inqld.com.au/business/2022/12/05/cryptos-brutal-reckoning-continues-as-local-exchange-swyftx-sacks-another-90-staff/
3.  https://www.afr.com/street-talk/crypto-exchange-swyftx-hunts-funding-flags-bridging-round-20221122-p5c0at

**Comment by molly** [comment]
I haven't been including companies who claim no exposure. It's possible that Swyftx is lying, of course, but without some more clarity into that I'm not planning to add them.
